api/src/main/java/org/eclipse/microprofile/rest/client/ext/package-info.java

```diff
@@ -19,7 +19,24 @@
  *******************************************************************************/
 
 /**
+ * APIs for extending MP Rest Client functionality - such as new providers.
+ * Example:
+ * <pre>
+ * public interface MyClientService {
+ *     @GET
+ *     @Path("/myService/{id}")
+ *     Widget getWidget(@PathParam("id") String id) throws UnknownWidgetException;
+ * }
+ * ...
+ * public class UnknownWidgetExceptionMapper implements ResponseExceptionMapper {
```

Comment:
Is it correct that a custom `ResponseExceptionMapper` doesn't require a `@RegisterProvider` annotation?  Seems odd that this can be figured out automatically but MBR, MBW, and RequestFilter require the anno.

Reply:
The `@RegisterProvider` annotation is not always required on the client interface - it would require that the mapper be registered using the `RestClientBuilder` API.  But since that is not obvious here, I'll update the interface to add the annotation.  Thanks!